docs/sidebars.js

```diff
@@ -64,6 +64,11 @@ module.exports = {
         }
       ],
     },
+    {
+      "community tools": [
+        "community-tools"
```

Comment:
Might want to consider creating a community tools folder if you're going to make a nested entry like this

If only the one page an no need for nesting or a folder, you can replace line 68 to70 with just the page ID 
`"community-tools"`